meta/main.yml

```diff
@@ -22,14 +22,17 @@ galaxy_info:
   - failover
 dependencies:
   - role: brianshumate.consul
-    when: patroni_dcs == "consul"
+    when: patroni_dcs == "consul" and
+          patroni_dcs_exist == false

```

Comment:
let's make this condition more pythonic and [PEP8 compliant](https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_true.html) by using `... and not patroni_dcs_exists`